hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -186,6 +190,10 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       if (tbl != null) {
         // If we try to create the table but the metadata location is already set, then we had a
         // concurrent commit
+        if (tbl.getTableType().equalsIgnoreCase(TableType.VIRTUAL_VIEW.name())) {
+          throw new AlreadyExistsException(
+              "View with same name already exists: %s.%s", tbl.getDbName(), tbl.getTableName());
+        }
```

Comment:
nit: newline

Reply:
Should we put this code before the comment?
The comment is for the next line below